web/js/index.js

```diff
@@ -1 +1,22 @@
-export { default as stub } from './stub';
+export { stub, stub2 } from './stub';
```

Comment:
I'm thinking maybe we should replace this file with the list of `import`s from the main.js file, and comment out the lines that aren't converted yet. That way we can make sure all of the references in the main.js imports are correct. I noticed a few that are named differently.